NuGet.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could you educate me on what the role of this file is? What problem occurs if it's not present?